src/pages/Article/ArticleContent/ArticleContent.tsx

```diff
@@ -166,38 +168,50 @@ const ArticleContent = ({ articleContent }: Props) => {
       <div className="article-info">
         <div className="author">
           <img
-            src={user?.profileImgUrl} // TODO: backend needs to add the user image
+            src={
+              author?.profileImgUrl || 'http://localhost:3000/src/components/NavBar/assets/logo.png'
+            } // TODO: backend needs to add the user image
             alt="author-avatar"
           />
           <p>
-            BY <Link to={`/user/${user?.id}`}>{user?.name}</Link>
+            BY <Link to={`/user/${author?.id}`}>{author?.name}</Link>
           </p>
         </div>
-        <div className="date">UPDATED: {updatedTime}</div>
-        <div className="date">POSTED: {createdTime}</div>
+        <div className="date">UPDATED: {nowToCreated(updatedTime)}</div>
+        <div className="date">POSTED: {nowToCreated(createdTime)}</div>
       </div>
       <ArticleMainContent mainContent={text} />
       <hr />
-      <div className="in-this-article">
-        <h3>In This Article</h3>
-        <ul className="game-list">
-          <li key={game?.id}>
-            <img
-              src={game?.imgUrl} // TODO: backend needs to add the game image url
-              alt="game-cover"
-            />
-            <p>{game?.name}</p>
-          </li>
-        </ul>
-      </div>
-      <div className="topics">
-        <h3>Topics</h3>
-        <ul className="tag-list">
-          {tagList.map((tag) => (
-            <li>{tag.name}</li>
-          ))}
-        </ul>
-      </div>
+      {
+        // if game is not null, show the game info
+        game && (
+          <div className="in-this-article">
+            <h3>In This Article</h3>
+            <ul className="game-list">
+              <li key={game?.id}>
+                <img
+                  src={
+                    game?.imgUrl || 'http://localhost:3000/src/components/NavBar/assets/logo.png'
+                  } // TODO: backend needs to add the game image url
+                  alt="game-cover"
+                />
+                <p>{game?.name}</p>
+              </li>
+            </ul>
+          </div>
+        )
+      }
+
+      {tagList && tagList.length > 0 && (
```

Comment:
```suggestion
      { tagList.length > 0 && (
```
上一级组件里面 initArticleContent 里面已经初始化了一个 tagList[ ], 所以这里的 tagList不起作用 可以删掉, 如果要加 也可以简化成
```suggestion
      { tagList?.length > 0 && (
```